scripts/scil_compute_NODDI.py

```diff
@@ -47,7 +48,7 @@ def _build_arg_parser():
     p.add_argument('in_bvec',
                    help='b-vectors filename, in FSL format (.bvec).')
 
-    p.add_argument('--in_mask',
+    p.add_argument('--mask',
                    help='Brain mask filename.')
     p.add_argument('--out_dir', default="results",
                    help='Output directory for the NODDI results. '
```

Comment:
Add the display of the default in the help. And possible add the check as suggested in FW (about `./`).